integration-tests/support/pages/tabs/LatestCommitsTabPage.ts

```diff
@@ -12,14 +13,35 @@ export class LatestCommitsTabPage {
   mergePR(owner: string, repoName: string, pullNumber: number, commitTitle: string, commitMessage: string) {
     const body = { commit_title: `${commitTitle}`, commit_message: `${commitMessage}` }
     Common.githubRequest('PUT', `https://api.github.com/repos/${owner}/${repoName}/pulls/${pullNumber}/merge`, body).then((result) => {
-      cy.wrap(JSON.stringify(result.body)).as(commitTitle)
+      expect(result.body.merged).to.be.true
+      Cypress.env(`${commitTitle}_SHA`, result.body.sha)
     });
   }
 
   editFile(gitRepo: string, filePath: string, commitMessage: string, updatedFileContentInBase64: string, fileSHA: string,) {
     const owner = gitRepo.split('/')[3];
     const repoName = gitRepo.split('/')[4];
     const body = { message: `${commitMessage}`, content: `${updatedFileContentInBase64}`, sha: `${fileSHA}` }
-    Common.githubRequest('PUT', `https://api.github.com/repos/${owner}/${repoName}/contents/${filePath}`, body)
+    Common.githubRequest('PUT', `https://api.github.com/repos/${owner}/${repoName}/contents/${filePath}`, body).then((result) => {
+      cy.log(`${commitMessage}_SHA : ${result.body.commit.sha}`)
```

Comment:
Should this log stay there?

Reply:
I think so, its beneficial to log the commit SHA at least here.